lib/hawkular/inventory/entities.rb

```diff
@@ -73,13 +73,21 @@ def initialize(res_hash)
     end
   end
 
-  class MetricType < BaseEntity
+  # Fields that are common for MetricType and Metric
+  module MetricFields
     # @return [String] GAUGE, COUNTER, etc.
     attr_reader :type
     # @return [String] metric unit such as NONE, BYTES, etc.
     attr_reader :unit
-    # @return [Long] collection interval in seconds
+    # @return [Long] collection interval in seconds, it has different semantics for MetricType and for Metric
```

Comment:
nit - missing a "." char, otherwise this and next lines are read together. I had to reread it to understand without punctuation